manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -5190,121 +5190,491 @@
+      Spalten erweitern, indem Sie ganz rechts in der Überschriftenzeile auf das kleine Dreieck
+      klicken und die gewünschte Information aus der Liste auswählen.
     </para>
 
-    <para>Each entry in the editor shows the details used by the security/currency:
+    <para>Die Bedeutung der einzelnen Spalten wird im <xref linkend="tool-commodity" /> beschrieben,
+      wohingegen zwei zusätzliche Spalten von &app; an Hand Ihrer Eingaben automatisch ausgefüllt
+      werden.
     </para>
 
-    <itemizedlist>
-      <listitem>
-        <para><emphasis role="bold">Type:</emphasis> Categories for organizing securities. &app; has the following
-          built in:
-          <itemizedlist>
-            <listitem>
-              <para><guilabel>CURRENCY</guilabel> or <guilabel>ISO4317</guilabel>: These are used for national
-                currencies and are not editable with the <guilabel>Security Editor</guilabel>.
-              </para>
-            </listitem>
+    <glosslist>
+      <glossentry id="tool-ge-PrintName">
+        <glossterm>Gedruckter Name</glossterm>
+        <glossdef>
+          <para>Der <guilabel>Gedruckte Name</guilabel> ist eine Kombination aus dem <guilabel>Symbol</guilabel> und
+            dem <guilabel>Namen</guilabel>.
+          </para>
+        </glossdef>
+      </glossentry>
+
+      <glossentry id="tool-ge-UniqueName">
+        <glossterm>Eindeutiger Name</glossterm>
+        <glossdef>
+          <para>Der <guilabel>Eindeutige Name</guilabel> setzt sich aus dem <guilabel>Namensraum</guilabel> (Typ)
+            und dem <guilabel>Symbol</guilabel> zusammen.
+          </para>
+        </glossdef>
+      </glossentry>
+    </glosslist>
 
-            <listitem>
-              <para><guilabel>FUND</guilabel>: Ordinarily used for open-ended mutual funds, i.e., those that one
-                purchases from and sells to only the issuing company and that are priced daily at
-                their net asset value.
-              </para>
-            </listitem>
+    <para>Im unteren Bereich des Wertpapier-Editors befinden sich Steuerelemente, deren Funktionsweise
+      nachfolgend beschrieben ist:
+    </para>
 
-            <listitem>
-              <para><guilabel>AMEX</guilabel>, <guilabel>ASX</guilabel>, <guilabel>EUREX</guilabel>,
-                <guilabel>NASDAQ</guilabel>, and <guilabel>NYSE</guilabel>: These represent a few of
-                the exchanges on which stocks, closed-end mutual funds, and exchange-traded funds
-                are traded.
+    <glosslist>
+      <glossentry>
+        <glossterm>Nationale Währungen anzeigen</glossterm>
+
+        <glossdef>
+          <para>Mit Aktivieren dieses Kontrollkästchens werden die Währungseinträge zu Ihrer Information
+            eingeblendet.
+            <note>
+              <para>Die nationalen Währungen sind gemäß ISO 4217 bezeichnet, und können nur eingeschränkt
```

Comment:
<ulink url="&url-wp-de;ISO_4217"/>

ggf. auch https://www.iso.org/iso-4217-currency-codes.html für technische Details. Natürlich zerlegen in 
entität url-iso und seite.